### ISSUE TITLE
show (last) failed jobs/schedules

### ISSUE THREAD

**Issue by rgroux** [erwindon/SaltGUI#180]
**Is your feature request related to a problem? Please describe.**
yes:
  - I need to know how many minion have the last run in a failed status
  - I wish i can have the date of the last run on a minion (and possibly the status)

**Describe the solution you'd like**
with the command ```salt-run jobs.last_run target='minion'``` we can extract the data
  - Result."minion".success -> True or False
  - StartTime -> date

**Describe alternatives you've considered**
Actually it s only via log on the server forwarded to a central log and then log analysis to extract th failed job, no information if the minion is stuck (or shutdown)
a solution is to have another job who check on every monion if it s ok or not.

**Additional context**
the output of the salt-run jobs.last_run target='server1'

```yaml
20190213111420601698:
    ----------
    Arguments:
    Function:
        state.highstate
    Minions:
        - server1
    Result:
        ----------
        server1:
            ----------
            out:
                highstate
            return:
                ----------
                <snip>
                ...
                </snip>
            success:
                True
    StartTime:
        2019, Feb 13 11:14:20.601698
    Target:
        dell2692dsy.paris.exalead.com
    Target-type:
        glob
    User:
        root
```

**Comment by erwindon** [comment]
This command returns only the last job for the given conditions. Since we already have the list of JIDs it is better to use `lookup_jid` instead. But the common factor is that the job details can only be retrieved one job at a time. thus still very slow.

**Comment by erwindon** [comment]
@rgroux @dawidmalina 
I've implemented something simple on the `Jobs` page.
A new column `Details` was added with the initial value `(click)`.
When you click it, the details for that job are retrieved and shown in a summary.
The data is not remembered when you leave this screen.

Any comments?

![afbeelding](https://user-images.githubusercontent.com/3663742/55667649-5a33ea80-585f-11e9-98c0-434f849df3df.png)
